app/flatpak-builtins-build.c

```diff
@@ -59,14 +59,49 @@ static GOptionEntry options[] = {
   { NULL }
 };
 
+static gboolean
+has_bind_mount_for_path (FlatpakBwrap *bwrap, const char *dest_path)
+{
+  guint i;
+
+  for (i = 0; i < bwrap->argv->len; i++)
+    {
+      const char *arg = g_ptr_array_index (bwrap->argv, i);
+
+      /* Check if this is a bind mount type: --bind, --ro-bind, or --ro-bind-data */
+      if (g_strcmp0 (arg, "--bind") == 0 ||
```

Comment:
these should handle the `-try` variants for this and for the one below and `--bind-data`.